ui/src/tracks/async_slices/index.ts

```diff
@@ -137,6 +143,33 @@ export class AsyncSliceTrack extends ChromeSliceTrack {
   static create(args: NewTrackArgs): Track {
     return new AsyncSliceTrack(args);
   }
+
+  drawChevron(ctx: CanvasRenderingContext2D, instant?: Instant) {
+    if (this.trackState.name.search(/^Buffer:? \d+/) !== 0) {
```

Comment:
Identifying a GPU buffer track by its name is obviously fragile. Since this isn't temporary code and hopefully Sokatoa will be around for a long time, can we come up with a better way?